concourse/model/traits/release.py

```diff
@@ -515,7 +515,12 @@ def process_pipeline_args(self, pipeline_args: JobVariant):
         # validate assets if present
         for asset in self.trait.assets:
             if not pipeline_args.has_step(asset.step_name):
-                raise ValueError(f'{asset=}\'s step_name refers to an absent build-step')
+                raise ValueError(
+                    f'{asset=}\'s step_name refers to an absent build-step. If the step in '
```

Comment:
stylistic nit: I would prefer using multiline-str-literals. will leave it up to you to change or keep as-is.